### ISSUE TITLE
How to Upgrade Resolve itself, within DaVinciBox?

### ISSUE THREAD

**Issue by FermataRest** [zelikos/davincibox#133]
How do I Upgrade Resolve itself, within DaVinciBox?
19.0.3_Linux is out, but I have no Idea how to execute the new .run from BlackMagic, within the Fedora Container, itself.

**Comment by zelikos** [comment]
> How do I Upgrade Resolve itself, within DaVinciBox? 19.0.3_Linux is out, but I have no Idea how to execute the new .run from BlackMagic, within the Fedora Container, itself.

tl;dr: same way you do a fresh installation

The simplest way would be to just run `setup.sh` on it like you're doing a fresh install; the existing davincibox container will still be used

Alternatively, you can follow the manual setup steps, but skip to the [Installing DaVinci Resolve](https://github.com/zelikos/davincibox#install-davinci-resolve) section since you already have the container.

**Comment by FermataRest** [comment]
Thanks for this!
Everything is upgraded!

tldr: sudo ./setup.sh DaVinci_Resolve_19.0.x_Linux.run
